apps/web/src/pages/get-started/consts/EchoUseCase.const.tsx

```diff
@@ -0,0 +1,110 @@
+import { StepDescription, StepText } from './shared';
+import { OnboardingUseCase } from './types';
+import { OnboardingUseCasesTabsEnum } from './OnboardingUseCasesTabsEnum';
+import { colors, Copy } from '@novu/design-system';
+import { Group, UnstyledButton, useMantineColorScheme } from '@mantine/core';
+import { useClipboard } from '@mantine/hooks';
+import { useSegment } from '@novu/shared-web';
+
+const COMMAND = 'npx novu-labs@latest echo';
+
+const CodeSnippet = () => {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** ‏Please move to a separate file -- this seems something that could be reusable with a couple props extracted‏